lib/Doctrine/Common/Lexer/AbstractLexer.php

```diff
@@ -302,6 +302,30 @@ protected function getModifiers()
     {
         return 'i';
     }
+    
+    /**
+     * Checks if given value was catched using getCatchablePatterns key
+     *
+     * @example getCatchablePatterns: ['[a-z]']
+     *          => isCatchedByKey('a', 0): true
+     *
+     * @example getCatchablePatterns: ['some_key' => '[a-z]']
+     *          => isCatchedByKey('c', 'some_key'): true
+     * @return bool
+     */
+    public function isCatchedByKey($value, $key)
+    {
+        if(!isset($this->getCatchablePatterns()[$key]) || !$value || !$this->input)
+        {
```

Comment:
Make the `{` in the same line as the if. Also, add a space between `if` and `(`.

Reply:
Done.